src/pages/AvailabilityPageSingle.js

```diff
@@ -136,7 +136,7 @@ const AvailabilityPageSingle = ({ offer, error, useChangeStatusMutation }) => {
             </Title>,
             <StatusForm
               key="reason-and-type"
-              offerType={offerType}
+              offerType={`${offerType}s`}
```

Comment:
This `s` can probably be removed?
```suggestion
              offerType={`${offerType}`}
```

Reply:
No this was needed, because in this component, the offerType that is picked from the api response is singular 😵‍💫